### PR TITLE
Javascript based is clearer than client side

### PR DESCRIPTION
**Pull request by bglusman**
language could probably use further fine tuning.  Final accepted version needs dupe to master also. @yaymukund @sethbannon @xcoderzach

### CODE REVIEW

File: _includes/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Getting Started
 =======
 
-EndDash is a bindings-aware client-side templating framework built on top of valid HTML.
+EndDash is a declarative two-way binding javascript based templating framework built on top of valid HTML.
```
Comment (yaymukund) [inline _includes/README.md]: How about "EndDash is a two-way binding javascript templating framework built on top of valid HTML"?

That's a lot of adjectives in a row. I sort of still prefer "bindings-aware Javascript templating framework." Those terms should all be familiar to anyone that is already researching this stuff. For the slogan, we want something catchy and conveys the gist. If it's confusing, we can expand later.

Comment (tobowers) [inline _includes/README.md]: we had at least 2 advanced JSers comment that they didn't know what "bindings-aware" means.

I like @yaymukund's rewrite there.  

alternatives:

EndDash is a javascript templating framework on top of valid HTML that allows two-way binding of Backbone objects to the DOM.

EndDash allows you to bind javascript objects to the DOM using semantic HTML.

Comment (bglusman) [inline _includes/README.md]: Yeah, mentioning Backbone's a probably a good idea in there... semantic HTML too.  Hmm... maybe top one with 'valid semantic HTML' ?

Comment (yaymukund) [inline _includes/README.md]: "valid, semantic" is redundant.

**Comment by sethbannon** [comment]
Final language: "EndDash is a two-way binding javascript templating framework built on top of semantic HTML"

**Pull request merged**